frontend/scss/app/components/dashboard/_status_card.scss

```
@@ -4,7 +4,12 @@
   gap: var(--gap-default);
   height: 100%;
   &__title {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    gap: var(--gap-small);
     color: var(--modest-text-color);
+    fill:  currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)